apps/console/src/features/identity-providers/components/wizards/organization-enterprise/organization-enterprise-authentication-provider-create-wizard.tsx

```diff
@@ -0,0 +1,484 @@
+                    identityAppsError.getErrorCode()) {
+                    setOpenLimitReachedModal(true);
+
+                    return;
+                }
+
+                if (error.response && error.response.data && error.response.data.description) {
+                    setAlert({
+                        description: t("console:develop.features.authenticationProvider.notifications." +
+                            "addIDP.error.description",
+                        { description: error.response.data.description }),
+                        level: AlertLevels.ERROR,
+                        message: t("console:develop.features.authenticationProvider.notifications." +
+                            "addIDP.error.message")
+                    });
+
+                    return;
+                }
+
+                setAlert({
+                    description: t("console:develop.features.authenticationProvider.notifications.addIDP." +
+                        "genericError.description"),
+                    level: AlertLevels.ERROR,
+                    message: t("console:develop.features.authenticationProvider.notifications.addIDP." +
+                        "genericError.message")
+                });
+            })
+            .finally(() => {
+                setIsSubmitting(false);
+            })
+        ;
+    };
+
+    /**
+     * Handles the final wizard submission.
+     *
+     * @param identityProvider - Identity provider data.
+     */
+    const handleWizardFormFinish = (identityProvider: IdentityProviderInterface): void => {
+
+        const connector: OutboundProvisioningConnectorInterface =
+            identityProvider?.provisioning?.outboundConnectors?.connectors[0];
+
+        const isGoogleConnector: boolean = get(connector,
+            IdentityProviderManagementConstants.PROVISIONING_CONNECTOR_DISPLAY_NAME) ===
+            IdentityProviderManagementConstants.PROVISIONING_CONNECTOR_GOOGLE;
+
+        // If the outbound connector is Google, remove the displayName from the connector.
+        if (connector && isGoogleConnector) {
+            delete connector[
+                IdentityProviderManagementConstants.PROVISIONING_CONNECTOR_DISPLAY_NAME
+            ];
+        }
+
+        delete (identityProvider.image);
+        delete (identityProvider.certificate);
+        delete (identityProvider.claims);
+        delete (identityProvider.provisioning);
+
+        // Use description from template.
+        identityProvider.description = template.description;
+
+        createNewIdentityProvider(identityProvider);
+    };
+
+    /**
+     * Called when modal close event is triggered.
+     */
+    const handleWizardClose = (): void => {
+
+        // Trigger the close method from props.
+        onWizardClose();
+    };
+
+    /**
+     * Close the limit reached modal.
+     */
+    const handleLimitReachedModalClose = (): void => {
+        setOpenLimitReachedModal(false);
```

Comment:
Add a new line.

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/3176